kubernetes/omsagent.yaml

```diff
@@ -660,6 +660,10 @@ spec:
             valueFrom:
                fieldRef:
                  fieldPath: spec.nodeName
+          - name: NODE_IP
```

Comment:
Are you also making a PR in the HELM repo and the AKS-RP repo's?

Reply:
> Are you also making a PR in the HELM repo and the AKS-RP repo's?
> These changes goes as part of release agent.